webapp/src/main/webapp/news-details-app/components/ExoNewsDetailsApp.vue

```diff
@@ -0,0 +1,46 @@
+<template>
+  <div id="newsDetail">
+    <div v-if="notFound" class="articleNotFound">
+      <i class="iconNotFound"></i>
+      <h3>{{ $t('news.app.noNews') }}</h3>
+      <v-btn class="btn btn-primary" @click="redirectUrl">{{ $t("news.composer.btn.backToHome") }}</v-btn>
+    </div>
+    <exo-news-details
+      v-else
+      :news="news"
+      :news-id="newsId" />
+  </div>
+</template>
+
+<script>
+export default {
+  props: {
+    newsId: {
+      type: String,
+      default: ''
+    },
+  },
+  data: () => ({
+    news: null,
+    notFound: false,
+  }),
+  created() {
+    this.$newsServices.getNewsById(this.newsId, false)
+      .then(news => {
+        if (news !== 'error') {
+          this.news = news;
+          if (!this.news.spaceMember) {
+            this.$root.$emit('restricted-space', this.news.spaceDisplayName);
+          }
+        } else {
+          this.notFound = true;
```

Comment:
can you test on news !== null here ? don't forget also that we should be in this case if the user is not allowed to see the news (not only news not existing)